api/src/test/java/com/messagebird/VoiceCallingTest.java

```diff
@@ -100,7 +101,7 @@ public void testGetLeg() throws IOException, GeneralException, UnauthorizedExcep
             VoiceLegStatus.Hangup, VoiceLegDirection.Outgoing,
             new BigDecimal("0.001519"), "EUR", 7,
             parseDate("2019-01-10T16:12:54Z"), parseDate("2019-01-10T16:13:54Z"),
-            parseDate("2019-01-10T16:13:24Z"), parseDate("2019-01-10T16:13:30Z")
+            parseDate("2019-01-10T16:13:24Z"), parseDate("2019-01-10T16:13:30Z"), SipResponseCode.OK
         );
```

Comment:
legFixture should be updated in this class, by adding SipResponseCode.OK 
`private static VoiceCallLeg legFixture =
        new VoiceCallLeg(
            LegIdFixture, CallIdFixture, "TestSource", "31612345678",
            VoiceLegStatus.Hangup, VoiceLegDirection.Outgoing,
            new BigDecimal("0.001519"), "EUR", 7,
            parseDate("2019-01-10T16:12:54Z"), parseDate("2019-01-10T16:13:54Z"),
            parseDate("2019-01-10T16:13:24Z"), parseDate("2019-01-10T16:13:30Z"),
                SipResponseCode.OK
        );`